src/utils/getDiscordIds.ts

```diff
@@ -0,0 +1,70 @@
+import { RDS_BASE_STAGING_API_URL } from "../constants/urls";
+import { UserBackend } from "../typeDefinitions/userBackend.types";
+import * as response from "../constants/responses";
+
+export const getDiscordIds = async (
+  userIds: string[] | string
+): Promise<string[] | string> => {
+  try {
+    const url = `${RDS_BASE_STAGING_API_URL}/users/userId`;
+
+    const numberOfBatches = Math.ceil(userIds.length / 6);
```

Comment:
Please move `6` to Constant. In the future if the CF increases/ decreases this number, we can change in one single place.

Reply:
That is highly unlikely that they change it from 6, but you are right, i ll still extract it into a constant